pkg/cloudprovider/plugins/aws/aws_ec2.go

```diff
@@ -98,50 +100,31 @@ func (ec2Cfg *ec2ServiceConfig) waitForInventoryInit(duration time.Duration) err
 	return backoff.Retry(operation, b)
 }
 
-// getInstanceResourceFilters returns filters to be applied to describeInstances api if filters are configured.
-// Otherwise, returns (nil, false). false indicates no selectors configured for the account and hence no cloud api needs
-// to be made for instance inventory.
-func (ec2Cfg *ec2ServiceConfig) getInstanceResourceFilters() ([][]*ec2.Filter, bool) {
-	var allFilters [][]*ec2.Filter
-
-	instanceFilters := ec2Cfg.instanceFilters
-	if len(instanceFilters) == 0 {
-		return nil, false
-	}
-
-	for _, filters := range ec2Cfg.instanceFilters {
-		// if any selector found with nil filter, skip all other selectors. As nil indicates all
-		if len(filters) == 0 {
-			return nil, true
-		}
-		allFilters = append(allFilters, filters...)
-	}
-	return allFilters, true
-}
-
-// getCachedInstances returns instances from the cache for the account.
-func (ec2Cfg *ec2ServiceConfig) getCachedInstances() []*ec2.Instance {
+// getCachedInstances returns instances from the cache applicable for the given selector.
+func (ec2Cfg *ec2ServiceConfig) getCachedInstances(selector *types.NamespacedName) []*ec2.Instance {
 	snapshot := ec2Cfg.resourcesCache.GetSnapshot()
 	if snapshot == nil {
-		awsPluginLogger().V(4).Info("Cache snapshot nil", "account", ec2Cfg.accountNamespacedName)
+		awsPluginLogger().V(4).Info("Cache snapshot nil",
+			"type", providerType, "account", ec2Cfg.accountNamespacedName)
```

Comment:
Since you are already in plugin, do you need provider type output in the log message?

Reply:
I am not sure, I saw it in azure plugin, wanted to keep it consistent. If not needed, should remove from everywhere.